specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/examples/ReimageVirtualMachine.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2018-10-01",
+    "vmName": "myVMName",
+    "parameters": {
+      "location": "West US",
```

Comment:
This isn't in the schema and is causing one of the model validation failures.

Reply:
Joel, is location parameter culprit?

Reply:
Yes I believe so.  Note that you can run this validation locally to verify the fix before updating the PR, see the docs [here](https://github.com/azure/oav).

Reply:
Thanks Joel for the tool to validate locally. I had updated the examples and validated it.

Reply:
However I see there are other validation errors coming for others unrelated to mine